src/components/InlineNotification/InlineNotification.tsx

```diff
@@ -0,0 +1,125 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './InlineNotification.module.css';
+import { Icon, Text } from '../../';
+
+export const VARIANTS = ['brand', 'success', 'warning', 'yield'] as const;
+
+const variantToIconAssetsMap: {
+  [key in 'brand' | 'success' | 'warning' | 'yield']: {
+    name: 'info' | 'check-circle' | 'warning' | 'yield';
+    title: 'info' | 'success' | 'alert' | 'review';
+  };
+} = {
+  brand: {
+    name: 'info',
+    title: 'info',
+  },
+  success: {
+    name: 'check-circle',
+    title: 'success',
+  },
+  warning: {
+    name: 'warning',
+    title: 'alert',
+  },
+  yield: {
+    name: 'yield',
+    title: 'review',
```

Comment:
Where did this title come from? I'm thinking this copy feels very specific when this component may be used in many different places. The other titles are more describing the level of importance the message has, and I feel like "review" is inconsistent.

Reply:
Sounds good, will title it `yield` instead